cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -403,13 +411,141 @@ include::../indexes/drop-an-index.asciidoc[leveloffset=+1]
 include::../indexes/drop-a-non-existing-index.asciidoc[leveloffset=+1]
 
 
+// TODO: should this section live here? or at some other level in the documentation
+[[indexes-future-indexes]]
+== Future indexes
+
+There are preparations for introducing two new types of indexes, point and range indexes.
+These together with text indexes will replace the current b-tree indexes.
+
+Like b-tree indexes, range indexes is created on one or more properties for all nodes or relationships with a given label or relationship type:
+
+* An index that is created on a single property for any given label or relationship type is called a _single-property index_.
+* An index that is created on more than one property for any given label or relationship type is called a _composite index_.
+
+The differences in the usage patterns between composite and single-property indexes described in <<administration-indexes-single-vs-composite-index>> also applies to range indexes.
+
+Point indexes, like text indexes, are a kind of single-property indexes, with the limitation that they only recognize properties point values.
+Nodes or relationships with the indexed label or relationship type where the indexed property is of another value type are not included in the index.
+
+=== Syntax
+
+.Syntax for managing future indexes
+[options="header", width="100%", cols="5a,3, 3a"]
+|===
+| Command | Description | Comment
+| [source, cypher, role=noplay]
+----
+CREATE RANGE INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName_1[,
+    n.propertyName_2,
+    …
+    n.propertyName_n])
+[OPTIONS "{" option: value[, ...] "}"]
+----
+| Create a range index on nodes, either on a single property or composite.
+
+Index provider can be specified using the `OPTIONS` clause.
+
+.4+.^| Best practice is to give the index a name when it is created.
+If the index is not explicitly named, it will get an auto-generated name.
+
+The index name must be unique among both indexes and constraints.
+
+The command is optionally idempotent, with the default behavior to throw an error if you attempt to create the same index twice.
+With `IF NOT EXISTS`, no error is thrown and nothing happens should an index with the same name, schema or both already exist.
```

Comment:
'the same name, schema or both' - type should probably be included here too?

Reply:
I seem to have missed this when I updated it in tests and for constraints at some point